torchao/prototype/awq/layout.py

```diff
@@ -0,0 +1,151 @@
+import torch
```

Comment:
can you use https://github.com/pytorch/ao/blob/378e6a8d6854d77efba45fcb1a4091724e9cfaa9/torchao/quantization/linear_activation_scale.py#L13 instead

Reply:
yea now that it is merged in, this class is not needed anymore I think